.github/workflows/continuous-integration.yml

```diff
@@ -36,15 +37,12 @@ jobs:
           - php-version: "5.3"
             dependencies: lowest
             experimental: false
-#          - php-version: "8.0"
-#            dependencies: highest
-#            experimental: false
-#          - php-version: "8.1"
-#            dependencies: lowest-ignore
```

Comment:
We have to drop this because it leads to installing dependencies broken on PHP 8.1.